core/storage/general_feedback/gae_models.py

```
@@ -381,6 +394,38 @@ def _validate_create_args(
                 'provided or both be None.'
             )
 
+    @classmethod
+    def _get_filtered_query(
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `_get_filtered_query`

(ANN206)